user_management/creating_and_reading_nodes/README.md

```diff
@@ -0,0 +1,65 @@
+
+# Creating and reading nodes with Memgraph Example 
+
+This example demonstrates how to use read and write queries via the `gqlalchemy` Python client.
+
+
+## 🧠 What This Example Does
+
+The script performs the following actions:
+
+1. **Connects to a running Memgraph instance** using `gqlalchemy`.
+2. **Creates an `Person` node** — creating a node with multiple properties.
```

Comment:
We don't need this step, as we're just showcasing execute_and_fetch()

Reply:
it's not needed since we're not using this class in any way. query works regardless